### PR TITLE
README update for com.android.support build issue

### PR DESCRIPTION
**Pull request by nicemanis**
This update adds the solution to the README mentioned [here](https://github.com/react-native-community/react-native-camera/issues/1086).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -450,34 +450,27 @@ To see more of the `react-native-camera` in action, you can check out the source
...
 ```
+#### A: As the error message hints `com.android.support:exifinterface:26.0.2` is already found in `com.android.support:support-v4:26.0.1`
+To fix this issue, modify your project's `android/app/build.gradle` as follows:
+```Gradle
+dependencies {
+    compile (project(':react-native-camera')) {
+        exclude group: "com.android.support"
+    }
 
-Add this to your AndroidManifest.xml:
```
Comment (sibelius) [inline README.md]: can we keep this lines? as well

Comment (nicemanis) [reply README.md]: I'm not sure if they are necessary as they didn't really fix the issue, at least for me.
The issue here is a conflict between two versions of `com.android.support:exifinterface`
But if there is different case where the changes to the manifest would fix the issue, then I can leave in also the manifest lines

Comment (nicemanis) [reply README.md]: I'm not sure if they are necessary as they didn't really fix the issue, at least for me.
The issue here is a conflict between two versions of `com.android.support:exifinterface`
But if there is different case where the changes to the manifest would fix the issue, then I can leave in also the manifest lines

Comment (sibelius) [reply README.md]: I think they are 2 different kind of problems

and we should provide solutions for both

Comment (nicemanis) [reply README.md]: Ok, then I will leave also those lines, I will also add this `exclude group: "com.google.android.gms"` line which is necessary if you use react-native-firebase

Comment (sibelius) [reply README.md]: tks

File: README.md [hunk 2]
```diff
@@ -450,12 +450,40 @@ To see more of the `react-native-camera` in action, you can check out the source
...
+      android:name=".MainApplication"
+      android:allowBackup="true"
+      android:label="@string/app_name"
+      android:icon="@mipmap/ic_launcher"
+      android:theme="@style/AppTheme"
+      tools:node="replace"
+    >
+```
+
 #### Q: When I try to build my project, I get following error:
```
Comment (sibelius) [inline README.md]: remove `Q:`

**Comment by sibelius** [comment]
@nicemanis great work on this, tks

**Comment by nicemanis** [comment]
@sibelius thanks

**Pull request merged**